stages/org.osbuild.ostree.config

```diff
@@ -63,13 +63,16 @@ def ostree(*args, _input=None, **kwargs):
 
 def main(tree, options):
     repo = os.path.join(tree, options["repo"].lstrip("/"))
-    config = options["config"]
+    sysroot_options = options["config"]["sysroot"]
```

Comment:
This is not a required property, right? So this needs to be `.get("sysroot", {})`?

Reply:
right.  thanks.  I only caught the nested bits.